rbac/common/base/base_state.py

```diff
@@ -247,46 +260,221 @@ def _find_in_state_container(self, container, address, object_id, target_id=None
         )
         return None
 
-    def get_from_state(self, state, object_id, target_id=None):
-        """Gets an address from the blockchain state
-        (state object is available when message is handled by transaction processor)"""
+    def deserialize(self, address, data):  # pylint: disable=unused-argument
+        """Deserialize the data of a blockchain address"""
+        # pylint: disable=not-callable
+        container = self._state_container()
+        container.ParseFromString(data)
+        return container
+
+    def get_address(self, context, address):
```

Comment:
Should this be called get_address_content() or get_address_data() per the docstring, or is it returning the deserialized address and the docstring is phrased strangely?

Reply:
get_address is a wrapper around get_address from the Sawtooth API

Reply:
Hmmm... if get_content_at_address(address) doesn't work then I will digress. Carrying on.